src/app/[locale]/_component/idpListOnApp/__test__/idpListOnApp.test.tsx

```diff
@@ -0,0 +1,25 @@
+import { useSearchParams } from 'next/navigation';
+import { screen } from '@testing-library/react';
+import * as it from '../../../../../dictionaries/it.json';
+import { IdpListOnApp } from '../idpListOnApp';
+import { renderWithProviders } from '@/app/[locale]/_utils/test-utils';
+// Mock the useSearchParams function
+jest.mock('next/navigation', () => ({
+  useSearchParams: jest.fn(),
+}));
+
+// Mock the behavior of useSearchParams
+const mockSearchParams = {
+  get: jest.fn(),
+};
+
+(useSearchParams as jest.Mock).mockReturnValue(mockSearchParams);
+
+describe('test suite for access error', () => {
+  test('should render "Generic error" when errorCode is null', async () => {
```

Comment:
Could we be more expressive here? Since we are testing the `IdpListOnApp` component, I suggest to put it also in the description to immediately spot where is the error in case of test failure.

Reply:
Sorry, it was a copy-paste error while copying the test template...